aleph0/index.ts

```diff
@@ -1,37 +1,57 @@
 import { Command } from "commander";
 import { z } from "zod";
-import { getProjectStructure, loadSnippets } from "./utils";
+import consola from "consola";
+import {
+  getKnowledge,
+  getKnowledgeForSnippet,
+  getProjectStructure,
+  loadSnippets,
+} from "./utils";
 import {
   createChangesArray,
-  findRelevantSnippets,
+  findRelevantSnippet,
```

Comment:
I changed the name of this to snippet. Because it only returns one. Will we ever need it to return multiple?

Reply:
Nope, this is good.